xcm/xcm_token_locations.json

```diff
@@ -243,22 +463,218 @@
         "interiors": {
           "v1": [
             {
-              "Parachain": 2090
+              "parachain": 2090
             },
             {
-              "GeneralIndex": 0
+              "generalIndex": 0
             }
           ],
           "v3": [
             {
-              "Parachain": 2090
+              "parachain": 2090
+            },
+            {
+              "generalIndex": 0
+            }
+          ]
+        }
+      },
+      {
+        "symbol": "RMRK",
+        "parents": 1,
+        "interiors": {
+          "v1": [
+            {
+              "parachain": 1000
+            },
+            {
+              "palletInstance": 50
+            },
+            {
+              "generalIndex": 8
+            }
+          ]
+        }
+      },
+      {
+        "symbol": "ZLK",
+        "parents": 1,
+        "interiors": {
+          "v1": [
+            {
+              "parachain": 2001
+            },
+            {
+              "generalKeyV1": "0x0207"
+            }
+          ]
+        }
+      },
+      {
+        "symbol": "vKSM",
```

Comment:
`vKSM` is a token in Kusama ecosystem. It must have `v3` version for `interiors` block

Reply:
Fixed